src/modules/configurator/components/ConfigureACar.tsx

```diff
@@ -1,11 +1,11 @@
 import { Routes, Route } from "react-router-dom";
-import { ConfigEdit } from "./ConfigEdit";
+import { ConfigDisplay } from "./ConfigDisplay";
 import { CarSelect } from "./CarSelect";
 
 export function ConfigureACar() {
   return (
     <Routes>
-      <Route path="configuration-edit/:id" element={<ConfigEdit />} />
+      <Route path="configuration-edit/query" element={<ConfigDisplay />} />
```

Comment:
Here also.

Reply:
Added configuration-edit/query to `configuratorRoutes`.